client/src/index.css

```diff
@@ -63,16 +73,37 @@ body {
 /* FAQ: The `.content` and `#react-root` boxes are only as tall as their content
         (which may be shorter than window height), can have siblings elements
         (would be influenced by force stretch), so gently stretch outward. */
-body,
-body > .content,
-#react-root {
+body {
+  /* As a flex container */
   display: flex;
   flex-direction: column;
 }
-body > .content,
-#react-root,
-/* FAQ: If the app root element has only a single child,
-        then it should be the same height as the app root */
-#react-root > :only-child {
+body > .content {
+  /* As a flex item */
+  flex-grow: 1;
+
+  /* As a flex container */
+  display: flex;
+  flex-direction: column;
+}
+#react-root {
+  /* As a flex item */
   flex-grow: 1;
+
+  /* As a stretched item container */
+  position: relative;
+}
+#react-root > :only-child {
+  /* As a stretched item */
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
+
+/* FP-546: If this style has become unnecessary, then remove it */
+.btn-link,
+.btn-link:hover {
+  color: var(--global-color-accent--normal);
 }
```

Comment:
**Why?** [Other components rely on this style](https://github.com/TACC/Frontera-Portal/pull/236#issuecomment-721198861) than those components whose stylesheets authored the style.